### ISSUE TITLE
Provide an archive with jar files you can just drop into the hot deploy folder of existing Opencast installations

### ISSUE THREAD

**Issue by fishfree** [opencast/annotation-tool#250]
I mean, I installed my Opencast by "sudo apt install opencast-7-allinone". I'm afraid if I build the Opencast from the source in the same server, the production environment will be damaged.

**Comment by JulianKniephoff** [comment]
Yeah, you definitely don't want to run two Opencasts to install the Annotation Tool. :wink:

There two basic possibilities basically:

- Install the tool into the `deploy` folder of your production install.

Looking at the setup of a random installation I'm maintaining, that is under `/usr/share/opencast/deploy`, so you could say

```
mvn <everything else you need> -DdeployTo=/usr/share/opencast
```

(Note that the corresponding Maven task will add the `/deploy` suffix itself.)

- Use the Karaf feature

This is more involved and I've been meaning to document this better, but time is a finite resource, unfortunately. x)

But what this basically amounts to is serving the Annotation Tool Artifacts somewhere (if you don't have any custom patches, you can even use the public Maven repo here on GitHub, which is also not documented yet, unfortunately :disappointed:), add that location as a Maven repository in your Opencast configuration and then add the Karaf feature to the boot features of whatever Opencast distribution you want to run the Annotation Tool on. You could even build a custom distribution, which I did for SWITCH for example, to have the Annotation Tool on its own server. But at that point you are leaving "RPM land". :wink:

**Comment by mtneug** [comment]
@JulianKniephoff maybe for those that install via a package manager, it would make sense to provide the JARs for download (e.g. via GitHub releases). Installation is then even simpler (1. download artefacts, 2. put them in the folder). I guess those that install via packages won't compile the annotation-tool either.

**Comment by JulianKniephoff** [comment]
@mtneug good idea actually. I'll see if I can add that to the CI setup some time soon. :thinking:

**Comment by fishfree** [comment]
@JulianKniephoff Thank you. This is my compiled annotation-tool JAR files
[annotation-tool-jar-files.zip](https://github.com/opencast/annotation-tool/files/3582348/annotation-tool-jar-files.zip)
 against to Opencast 7.2.

**Comment by JulianKniephoff** [comment]
While clean up a bit here, I just noticed this can be closed in favor of #178.
